docs/update-locking.md

```diff
@@ -2,7 +2,7 @@
 
 Locking updates means that the Supervisor will not be able to kill your application. This is meant to be used at critical sections of your code where you don't want to be interrupted, or to control that updates are only installed at certain times.
```

Comment:
NIT: `or to control` would read better as `or to ensure`

Reply:
Outside of update scope, but updated because it's nicer :)